app/src/main/java/ch/epfl/sdp/blindly/database/UserRepositoryImpl.kt

```diff
@@ -181,24 +181,49 @@ class UserRepositoryImpl constructor(
 
 
     /**
-     * Removes another liked or matched user from current user.
+     * Removes a matched user from likes and matches and adds it to dislikes.
      *
-     * @param field field to remove a User (either from LIKES or MATCHES)
      * @param userId current user's ID
      * @param matchId matched user's ID
      */
-    override suspend fun removeMatchFromAUser(field: String, userId: String, matchId: String) {
-        var updatedList: ArrayList<String>? = arrayListOf()
+    override suspend fun removeMatchFromRemovingUser(
```

Comment:
I think the previous name of the function was clearer, I'm not sure what a RemovingUser is... :O

Reply:
Yeah since we remove match from the remote user as well, I tried to make it more explicit but I guess it was confusing/ I will rename it to removeMatchFromCurrentUser which should be clearer.

Reply:
Perfect, thanks! :D